mne/utils/docs.py

```diff
@@ -2262,6 +2262,29 @@
     fail irrespective of this parameter.
 """
 
+# Export
+docdict['export_warning'] = """
+.. warning::
+    Since we are exporting to external formats, there's no guarantee that all
+    the info will be preserved in the external format. To save in native MNE
+    format (``.fif``) without information loss, use :func:`save` instead.
+"""
+docdict['export_params_base'] = """
+Parameters
+----------
+fname : str
+    Name of the output file.
+"""
+docdict['export_params_fmt'] = """
```

Comment:
```suggestion
docdict['export_params_fmt'] = """
fmt : 'auto' | 'eeglab'
```

Reply:
The idea is raw and epochs might have different supported formats in the future. So I intentionally left the supported formats out of the docdict. What do you think?